common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -729,29 +729,30 @@ def available_translations(self, transcripts, verify_assets=None, include_val_tr
 
         sub, other_langs = transcripts["sub"], transcripts["transcripts"]
 
+        # include val transcripts
```

Comment:
do we need `include_val_transcripts` parameter?

Reply:
nope. will remove this.